utils/apptainer/build.sh

```diff
@@ -82,7 +83,7 @@ main() {
   images+=("build/openjdk-17")
   images+=("bcftools-1.14")
   images+=("annotsv-3.2.3")
-  images+=("capice-5.0.0")
+  images+=("capice-5.1.0")
```

Comment:
Due to a wontfix for https://trac.molgeniscloud.org/ticket/9808 the gnomAD HN feature should be removed from https://github.com/molgenis/capice/releases/tag/v5.1.0? And this release should not be used in VIP?

Reply:
CAPICE 5.1.0 is more than just the gnomAD_HN feature. Since the release of CAPICE 5.0.0 each capice model created with major version 5 is compatible with CAPICE major version 5, which means that the use of the model 5.0.0-v1 should still be fully operational with CAPICE 5.1.0